src/components/Todo.jsx

```diff
@@ -1,6 +1,9 @@
 import {useState} from "react";
 import {useSelector} from "react-redux";
 import {operationsReducer} from "../redux/todo/reducers/operations.js";
+import Icon, {icon} from "react-icons-kit"
+import {trash} from "react-icons-kit/feather/trash"
```

Comment:
cant we use just fontawesome icons?

Reply:
tried them first, werent working somehow, so moved to this